Assets/Scripts/Actors/ActorPhysics.cs

```diff
@@ -72,7 +71,11 @@ public float rollMoveSpeed
 
 	public float normalizedMoveSpeed
 	{
-		get { return _rigidbody.velocity.magnitude / sprintMoveSpeed; }
+		get
+		{
+			Vector3 horizontalVelocity = _rigidbody.velocity.SetY( 0.0f );
+			return horizontalVelocity.magnitude / sprintMoveSpeed;
```

Comment:
Shouldn't this be current max speed? So if walking, then normalized speed would be based on walking max speed?